tests/zfs-tests/tests/functional/snapshot/rollback_003_pos.ksh

```diff
@@ -101,11 +107,4 @@ log_must touch /$TESTPOOL/$TESTFILE/$TESTFILE.1
 
 log_must zfs rollback $SNAPPOOL.1
 
-#
-# Workaround for issue #6143.  Issuing a `df` seems to properly force any
-# negative dcache entries to be invalidated preventing subsequent failures
-# when accessing the mount point.  Additional investigation required.
-#
-log_must df
-
```

Comment:
This also seems to be a revert of another commit?

Reply:
not my patch. never touched this file